common/lcmInputs.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import "time"
+
+type InputMsgType uint16
+
+var EInputMsgType InputMsgType
+
+func(InputMsgType) Invalid()                                InputMsgType {return InputMsgType(0)}
+func(InputMsgType) CancelJob()                              InputMsgType {return InputMsgType(1)}
+func(InputMsgType) E2EInterrupts()                          InputMsgType {return InputMsgType(2)}
+func(InputMsgType) ThroughputAdjustment()                   InputMsgType {return InputMsgType(3)}
+
+
+var MsgTypeMap = map[string]InputMsgType{
```

Comment:
please check the other enums, they have a different pattern to achieve the "fromString" functionality